regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -107,6 +108,10 @@ var CommentReviewView = Backbone.View.extend({
     this.submitMode();
   },
 
+  back: function() {
+    window.history.back();
+  },
```

Comment:
What if a user opened the review page in a new tab, or navigated to the url directly? Is back always the right behavior?